src/box/sql/vdbe.c

```diff
@@ -467,12 +467,110 @@ int sqlVdbeExec(Vdbe *p)
 	}
 #endif
 
+#if defined(SQL_DEBUG)
+
+#define VDBE_TRACE() \
```

Comment:
I do not clearly understand why you introduced this macro..Why not simply inline/put it into function?
Macros almost always are considered to be evil (hard to read&patch)..

Reply:
Fixed.